src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt

```diff
@@ -102,21 +104,20 @@ class MastodonSidePlugin : DynamicCommand() {
             ) ++chosenLevel
             //NB: theoretically the channel should be always found...
             if (chosenLevel == levelNames.size) return  // !?
-            logService!!.info(
-                "Selected volume from channel " + channelIdx + " of "
-                        + (chosenLevel + 1) + "-best resolution level..."
+            logger.info(
```

Comment:
so `logService` is actually dead here.... so we could then remove L60, no?

Reply:
You are right, I think we can remove it here.